src/core/metrics/aggregate.go

```diff
@@ -59,11 +59,13 @@ func SaveCollections(metricsCollections Collections, reports ...*proto.MetricsRe
 				}
 			}
 
-			for _, simpleMetric := range stats.Simplemetrics {
-				if metrics, ok := metricsCollections.Data[dimensionsChecksum].RunningSumMap[simpleMetric.Name]; ok {
-					metricsCollections.Data[dimensionsChecksum].RunningSumMap[simpleMetric.Name] = metrics + simpleMetric.GetValue()
-				} else {
-					metricsCollections.Data[dimensionsChecksum].RunningSumMap[simpleMetric.Name] = simpleMetric.GetValue()
+			if stats.GetSimplemetrics() != nil {
```

Comment:
would it be cleaner to assign a variable here? and reuse it in the range on the line below?